effekt/shared/src/main/scala/effekt/core/Transformer.scala

```diff
@@ -117,18 +117,19 @@ object Transformer extends Phase[Typechecked, CoreTransformed] {
     case f @ source.ExternDef(pure, id, _, vps, bps, _, bodies) =>
       val sym@ExternFunction(name, tps, _, _, ret, effects, capt, _) = f.symbol
       assert(effects.isEmpty)
+      assert(bodies.length == 1, "Externs are resolved before core")
```

Comment:
The `.head` below is a bit hidden. Maybe refactor to

```
val tBody = bodies match {
  case source.ExternBody.StringExternBody(ff, body) :: Nil => ...
  case _ => Context.panic("Externs should be resolved and desugared before core.Transformer")
}
```
?

Reply:
Good idea.